src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -44,7 +44,7 @@
  * <li>groups static imports: ensures that static imports are at the top (or the
  * bottom) of all the imports, or above (or under) each group, or are treated
  * like non static imports (@see {@link ImportOrderOption}</li>
- * </ul>
+ * </ul>.
```

Comment:
Since this is becomming connected to `SummaryJavadoc`, please add a UT to `SummaryJavadoc` so we can have some documentation of this.

@ps-sp Missing a change in grammar almost got by unnoticed on changes in checks. Nothing picked this up but this one little change and it wasn't obvious at all.
I suggest you do regression on all javadoc checks with any ANTLR changes from now on to make sure this doesn't happen anywhere else.